### DISCUSSION TITLE
Tensor element ordering when converting to ANTs from fsl

### DISCUSSION THREAD

**Discussion by aplumley** [ANTsX/ANTs#1443] [Q&A]
Hello,

I am processing some DTI images and followed the instructions on the page **_Importing diffusion tensor data from other software_** to convert from FSL into ANTs format. 

The header information all looks correct, but the ordering of the elements after using ImageMath ComponentTo3DTensor appears to be in upper-triangular, rather than the lower-triangular that ANTs needs. Is there a simple way to change the order? 

Thanks in advance,

Alix

**Comment by cookpa** [comment]
The ordering is handled by ITK and depends on the output file format. We would need a complete reproducible example with code and data to diagnose any problems.
